### PR TITLE
Add method on ProjectsHandler to get event logs

### PR DESCRIPTION
**Pull request by sonhal**
I need to be able to get the event logs for a given project ref: https://api.aiven.io/doc/#operation/ProjectGetEventLogs.

### CODE REVIEW

File: project.go [hunk 1]
```diff
@@ -81,6 +81,19 @@ type (
 		APIResponse
 		Projects []*Project `json:"projects"`
 	}
+
```
Comment (ivan-savciuc) [inline project.go:84]: Please add annotations to make `gofmt` happy

File: project.go [hunk 2]
```diff
@@ -81,6 +81,19 @@ type (
...
+	ProjectEventLogEntries struct {
+		APIResponse
+		Events []*ProjectEvent `json:"events"`
+	}
+
+	ProjectEvent struct {
```
Comment (ivan-savciuc) [inline project.go:90]: Same here, add `//ProjectEvent ...`

File: project.go (hunk 1 above)
Comment (ivan-savciuc) [reply project.go:84]: Also, please add Response -> `ProjectEventLogEntriesResponse` to the structure name, to make it constant with the rest of the code.

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
Thanks for contributing! Aside from these minor things, can you please add this functionality to the project acceptance test?

File: project.go (hunk 1 above)
Comment (sonhal) [reply project.go:84]: Fixed in comit 848b4bf

File: project.go (hunk 2 above)
Comment (sonhal) [reply project.go:92]: Fixed in 848b4bf

**Comment by sonhal** [comment]
Please review @ivan-savciuc :)

File: project_acc_test.go [hunk 3]
```diff
@@ -72,7 +72,19 @@ var _ = Describe("Projects", func() {
...
+			Expect(logErr).To(BeNil())
+			Expect(events).ToNot(BeNil())
+			Expect(events).Should(Not(BeEmpty()))
+			for _, event := range events {
+				Expect(event).NotTo(BeNil())
+			}
+		})
+	})
+
+		AfterEach(func() {
```
Comment (ivan-savciuc) [inline project_acc_test.go:87]: Here indentation is unnecessary

**Review by ivan-savciuc — CHANGES_REQUESTED** [review]
Thanks for adding the test to the PR :+1: 

Found a small indentation issue. Otherwise, everything is looking good. 

Also, could to squash all changes to one commit?

File: project_acc_test.go (hunk 3 above)
Comment (sonhal) [reply project_acc_test.go]: Fixed and squashed

**Review by ivan-savciuc — APPROVED** [review]

**Comment by sonhal** [comment]
Merge?

**Pull request merged**